### PR TITLE
Updated clustering documentation

### PR DESCRIPTION
**Pull request by mfojtik**
@bparees PTAL

### CODE REVIEW

File: 2.4/examples/replica/README.md [hunk 1]
```diff
@@ -15,8 +15,7 @@ MongoDB’s recommended replication strategy.
 In order to run this Docker image, you need to have the latest version of
 OpenShift Origin v3.
 
-* `openshift start --latest-images &> server.log &`
-* `oc create -f ./2.4/examples/replica/mongodb_controller.json`
+* `oc create -f ./2.4/examples/replica/mongodb_replica_set.json`
```
Comment (bparees) [inline 2.4/examples/replica/README.md]: tell people what this is doing/creating.

File: 2.4/examples/replica/README.md [hunk 2]
```diff
@@ -25,38 +24,39 @@ OpenShift Origin v3.
...
-This Pod serves as an replSet initialization member. Inside the
-'initiate_replica.sh' command is ran, which will wait for all members created by
-the ReplicationController to become available and then creates the initial
-MongoDB replica set. This set is then distributed to all members.
-This Pod is a 'run-once' Pod, which means that once the replSet is initialized
-that the members receive the replication data, this Pod is destroyed.
+This pod is responsible for initializing the MongoDB replica set. It runs just
+once when the resources are being created for the first time. It also creates
+the initial database and users. After the initialization is completed and data
+are replicated to other member of MongoDB replication set, this pod gave up the
```
Comment (bparees) [inline 2.4/examples/replica/README.md]: s/gave/gives/

File: 2.4/examples/replica/README.md [hunk 3]
```diff
@@ -25,38 +24,39 @@ OpenShift Origin v3.
...
+PRIMARY role in the replication set and shutdown.
```
Comment (bparees) [inline 2.4/examples/replica/README.md]: s/replication/replica/ ?

Comment (jhadvig) [inline 2.4/examples/replica/README.md]: most of the time we are using the `replication` but on line 35 we use `replica`. Would be nice to unify it.

**Comment by mfojtik** [comment]
@bparees thanks! updated.

File: 2.4/examples/replica/README.md [hunk 4]
```diff
@@ -25,38 +26,39 @@ OpenShift Origin v3.
...
-### ReplicationController 'mongodb'
+### DeploymentConfig 'mongodb'
 
-This resource defines the PodTemplate of the MongoDB replica member. Each member
-is started as a MongoDB server without replication data. Once the member is up,
-the container will advertise this member to the MongoDB replica PRIMARY member
-and add it into the cluster.
-When this member is removed, the container, before it exits will properly remove
-the member from existing MongoDB cluster.
+This resource defines the replication controller template that manage the
```
Comment (bparees) [inline 2.4/examples/replica/README.md]: s/manage/manages/

File: 2.4/examples/replica/README.md [hunk 5]
```diff
@@ -72,11 +74,10 @@ For the 'mongodb-service' Pod, you have to set following variables:
 * `MONGODB_PASSWORD` - the regular MongoDB user password (Generated by default).
 * `MONGODB_KEYFILE_VALUE` - value for 'keyFile[1](http://docs.mongodb.org/manual/tutorial/generate-key-file)' file that MongoDB members use for authorization internally (Generated by default).
 
-All the expressions, from which values of the environment variables are generated 
-are defined in `mongodb_controller.json`
+All the expressions, from which values of the environment variables are generated
```
Comment (bparees) [inline 2.4/examples/replica/README.md]: generated,

**Comment by bparees** [comment]
2 final nits and lgtm.

**Comment by mfojtik** [comment]
@bparees fixed.

**Pull request merged**